handlers/main.yml

```diff
@@ -1,26 +1,26 @@
 ---
 - name: restart yum-cron
-  service:
+  systemd:
     name: yum-cron
     state: restarted
     enabled: yes
 
 - name: start haveged
-  service:
+  systemd:
     name: haveged
     state: started
     enabled: yes
 
 - name: reload systemd
-  command: systemctl daemon-reload
+  systemd:
```

Comment:
empty command. i suppose it should look like this:   'systemd: daemon_reload=yes'